src/caches/abstractbeliefpropagationcache.jl

```diff
@@ -24,26 +25,48 @@ function data_graph_type(bpc::AbstractBeliefPropagationCache)
 end
 data_graph(bpc::AbstractBeliefPropagationCache) = data_graph(tensornetwork(bpc))
 
-function default_message_update(contract_list::Vector{ITensor}; normalize=true, kwargs...)
-  sequence = contraction_sequence(contract_list; alg="optimal")
-  updated_messages = contract(contract_list; sequence, kwargs...)
+function message_update_function(
```

Comment:
```suggestion
function message_update(
```
Just simpler and I don't think `_function` is needed now that we're using `Algorithm` more.